docs/classical_control.ipynb

```diff
@@ -147,7 +158,27 @@
     "    cirq.X(bob).with_classical_controls('A'),\n",
     "    cirq.Z(bob).with_classical_controls('M'),\n",
     ")\n",
-    "print(circuit)"
+    "print(circuit)\n",
+    "\n",
+    "# Simulate the message and teleport circuits for bloch vectors\n",
```

Comment:
Capitalize Bloch outside variable names.

Also, why are we using Bloch vectors here? I guess `bloch_vector_from_state_vector` is convenient for comparing individual qubits out of the entire state, but it still feels kind of roundabout.

Reply:
That's exactly why. It's a slight trade-off of adding more code so that there is more direct printed content indicating that the two qubits are in the same state. The final_state_vectors by themselves are not directly comparable and just showing counts from results could obfuscate the point because different states could produce the same results. Is there a better way of seeing the state of a single qubit?